models/api.ts

```diff
@@ -179,3 +183,14 @@ async function getGovernanceAccountsImpl<TAccount extends GovernanceAccount>(
 
   return accounts
 }
+
+export async function getGovernanceAccount<TAccount extends GovernanceAccount>(
+  connection: Connection,
+  accountPubKey: PublicKey,
+  accountClass: GovernanceAccountClass
+) {
+  const accountInfo = await connection.getAccountInfo(accountPubKey)
+  const account = BorshAccountParser(accountClass)(accountPubKey, accountInfo)
+
+  return account as ParsedAccount<TAccount>
+}
```

Comment:
hah! that's a nice one, i'll use it as well :)

Reply:
:)